### DISCUSSION TITLE
Feature Request: Organization Level Custom Metadata

### DISCUSSION THREAD

**Discussion by AnthonyHerman** [community/community#51990] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Custom Metadata at the organization level would be extremely useful for large Enterprises with hundreds of internal organizations.

**Comment by beingminimal** [comment]
@AnthonyHerman not only for large enterprises, but for them also who are managing many repos in one free org also. so please consider all plan's orgs. thanks
